parts/linux/cloud-init/artifacts/aks-check-network.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+# 
+# AKS Check Network
+#
+# This script is used to check network connectivity from the node to certain required AKS endpoints 
+# and log the results to the events directory. For now, this script has to be triggered manually to
+# collect the log. In the future, we will run it periodically to check and alert any issue.
```

Comment:
we can setup a systemd timer to run this periodically later if we want

Reply:
yes! that's what I plan to do in the future. Still need to decide how often we want to run it